src/stylesheets/themes/hugo-terminal-orange/index.scss

```diff
@@ -0,0 +1,24 @@
+@import "./variables";
+@import "../../index";
+@import "./syntax";
+@import "./button.scss";
+@import "./glow.scss";
+@import "./heading.scss";
+@import "./comment.scss";
+@import "./selection.scss";
+@import "./placeholder.scss";
+@import "./combobox.scss";
+@import "./code.scss";
+
+body {
+  background-color: $bg-page;
+}
+
+iframe {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
```

Comment:
what is this rule for?

Reply:
Do not know, I copied it from `github-dark-orange` theme. I deleted it and it seems to work fine without it.